lumium-space/src/hooks/api/useApi.ts

```diff
@@ -1,17 +1,17 @@
 import { SECURE, V1 } from "@routes/api/v1";
-import { AUTH, AUTH_SIGNIN } from "@routes/space";
+import { AUTH, AUTH_SIGNIN, ROOT } from "@routes/space";
 import axios, { AxiosInstance } from "axios";
 import Router from "next/router";
 
-const instance: AxiosInstance = axios.create({ baseURL: process.env.NEXT_PUBLIC_API_HOST ? process.env.NEXT_PUBLIC_API_HOST + V1 : "", withCredentials:true });
+const instance: AxiosInstance = axios.create({ baseURL: process.env.NEXT_PUBLIC_API_HOST ? process.env.NEXT_PUBLIC_API_HOST + V1 : "", withCredentials: true });
 instance.interceptors.response.use(
     (r) => r,
-        (error) => {
+    (error) => {
         if (!error.response) {
             return Promise.resolve(error);
         }
         if (error.response.status === 401) {
-            if (!Router.asPath.startsWith(AUTH)) {
+            if (!Router.asPath.startsWith(AUTH) && Router.asPath != ROOT) {
```

Comment:
im getting redirected on landing page if im not logged in which shouldnt be

Reply:
ah, i see. yes, this can happen because you still have an old token signed with an old token signature secret, which leads to invalid credentials being sent to the api. good catch